docs/developer-guide/selectors.md

```diff
@@ -131,3 +131,7 @@ Or you can enforce that calls to `setTimeout` always have two arguments:
 ```
 
 Using selectors in the `no-restricted-syntax` rule can give you a lot of control over problematic patterns in your codebase, without needing to write custom rules to detect each pattern.
+
+### Known issues
+
+Due to a [bug](https://github.com/estools/esquery/issues/68) in [esquery](https://github.com/estools/esquery), regular expressions that contain a forward-slash character `/` aren't porperly parsed, so `[value=/some\/path/]` will be a syntax error. As a [workaround](https://github.com/estools/esquery/issues/68), you can replace the `/` character with its unicode counterpart, like so: `[value=/some\\u002Fpath/]`.
```

Comment:
`\\` in `[value=/some\\u002Fpath/]` might be confusing. Selector itself should have only one `\`, while the other `\` is required only if the selector is represented in a string literal?

Reply:
I'd suggest only `\` here (like it is in `[value=/some\/path/]`), and an example that disallows import from `foo/bar`:

```json
{
  "rules": {
    "no-restricted-syntax": ["error", "ImportDeclaration[source.value=/foo\\u002Fbar/]"]
  }
}
```